.chronus/changes/fix-common-types-gen-2024-6-3-16-15-21.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
As discussed, can you move x-ms-identifier from types.tsp into the supplement file.